src/pages/invoices/create/Create.tsx

```diff
@@ -163,7 +163,7 @@ export function Create() {
           onChange={(id) => handleChange('client_id', id)}
           onClearButtonClick={() => handleChange('client_id', '')}
           onContactCheckboxChange={handleInvitationChange}
-          readonly={searchParams.get('table') === 'tasks'}
+          readonly={searchParams.get('project') === 'true'}
```

Comment:
Why?

Reply:
David and I had a lot of discussions on Slack about this functionality, so we realized that we always made this selector read-only if we invoiced a task. So David recommended that I make this selector read only when the task has a project selected. It should override all client_id values. So I pass it as a URL parameter and check it here.

Reply:
@beganovich this is correct. Selecting the project will always override any other selector that contains a reference to a client_id.